Medidata.Cloud.ExcelLoader/CellStyleProviders/ExtractedCellStyleProvider.cs

```
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Spreadsheet;
+
+namespace Medidata.Cloud.ExcelLoader.CellStyleProviders
+{
+    /// <summary>
+    /// Extract StyleSheet from an exisiting SpreadSheet and use 'Output' and 'Normal' style
+    /// </summary>
+    public class ExtractedCellStyleProvider: ICellStyleProvider
+    {
+        private uint headerStyleIndex = 0;
+        private uint textStyleIndex = 0;
+        public uint GetHeaderStyleIndex()
+        {
+            return headerStyleIndex;
```

I think we should throw an exception if headerStyleIndex is 0, so as to notify user to run `AttachTo` before calling `GetHeaderStyleIndex` and `GetTextStyleIndex`, like

```cs
public uint GetHeaderStyleIndex() {
    if(headerStyleIndex==0) throw new Exception("Didn't attach to any document yet.");
    return headerStyleIndex;
}
```